app/main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def die(self) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
+        )
+
+    @classmethod
+    def __str__(cls) -> str:
```

Comment:
The `__str__` method is incorrectly decorated with `@classmethod`. The `__str__` method should not be a class method, as it is intended to provide a string representation of an instance, not the class itself. Consider removing the `@classmethod` decorator.